randovania/games/prime1/logic_database/Phazon Mines.txt

```diff
@@ -495,8 +499,16 @@ Extra - aabb: [40.680916, 162.93446, 9.550913, 70.02207, 207.09828, 75.1311]
           Morph Ball
           Any of the following:
               Spider Ball
-              # https://youtu.be/X-6LeSNC5LM?t=408
-              Morph Ball Bomb and Space Jump Boots and Bomb Space Jump (Advanced) and Standable Terrain (Expert) and Shoot Wave Beam
+              All of the following:
+                  # https://youtu.be/X-6LeSNC5LM?t=408
+                  Space Jump Boots and Standable Terrain (Expert)
+                  Any of the following:
+                      Morph Ball Bomb and Bomb Space Jump (Advanced)
+                      # https://youtu.be/PjXSkYQXDls
+                      Boost Ball and Wall Boost (Hypermode)
```

Comment:
Are you *sure* this is NSJ? I thought that vid was just a speed strat.

Also IUJs should use whatever the HoTE IUJ trick requirements are

Reply:
Not sure what you mean by NSJ here. This has SJ and is under `All of the following:`.